test/performance/benchmarks/scale-from-zero/dev.config

```diff
@@ -0,0 +1,36 @@
+# Creating this benchmark:
+# mako create_benchmark \
+#   test/performance/benchmarks/scale-from-zero/continuous/dev.config
+# Updating this benchmark:
+# mako update_benchmark \
+#   test/performance/benchmarks/scale-from-zero/continuous/dev.config
+project_name: "Knative"
+benchmark_name: "Development - Serving scale from zero"
+description: "Scale from zero test against ksvcs in parallel."
+benchmark_key: '5024954898710528'
+
+# Human owners for manual benchmark adjustments.
+owner_list: "mattmoor@google.com"
+owner_list: "vagababov@google.com"
+owner_list: "chizhg@google.com"
+owner_list: "yanweiguo@google.com"
+
+# Only this robot should publish data to Mako for this key!
```

Comment:
This is dev, so I don't think that's right.
Can you copy the dataplane robots here?

Reply:
Done.